metron-platform/metron-writer/src/main/java/org/apache/metron/writer/FlushPolicy.java

```diff
@@ -40,10 +39,12 @@
   boolean shouldFlush(String sensorType, WriterConfiguration configurations, int batchSize);
 
   /**
-   * This method is used to clear any internal state a {@link org.apache.metron.writer.FlushPolicy} maintains to determine if a batch should be flushed.
-   * This method is called for all {@link org.apache.metron.writer.FlushPolicy} implementations after a batch is flushed with
-   * {@link org.apache.metron.writer.BulkWriterComponent#flush(String, BulkMessageWriter, WriterConfiguration, List)}.
-   * @param sensorType
+   * Allows a policy to define what should happen when a flush occurs.
+   *
+   * <p>This method is called immediately after a batch has been flushed.
+   *
+   * @param sensorType The type of sensor generating the messages
+   * @param response A response containing successes and failures within the batch
    */
-  void reset(String sensorType);
+  void onFlush(String sensorType, BulkWriterResponse response);
```

Comment:
We just needed to expand the `reset` method a bit.  So now its called `onFlush` and includes the `BulkWriterResonse`.